lib/server/limiter.js

```diff
@@ -5,17 +5,19 @@ const errors = require('storj-service-error-types');
 
 const tmpl = '{"rate_limited": {"url": "%s", "method": "%s", "ip": "%s"}}';
 
-module.exports.DEFAULTS = {
-  lookup: function(req) {
-    return [req.headers['x-forwarded-for'] || req.connection.remoteAddress];
-  },
-  onRateLimited: function(req, res, next) {
-    log.info(tmpl,
-             req.originalUrl,
-             req.method,
-             req.headers['x-forwarded-for'] || req.connection.remoteAddress);
-    return next(new errors.RateLimited('Too Many Requests'));
-  },
-  total: 1000,
-  expire: 1000 * 60 // 1 minutes
+module.exports.DEFAULTS = (config={}) => {
+  return {
+    lookup: function(req) {
+      return [req.headers['x-forwarded-for'] || req.connection.remoteAddress, req.route.path];
```

Comment:
We could use both path and method: `req.path, req.method`, as similar to: https://github.com/Storj/service-middleware/blob/master/lib/rate-limiter.js#L33